index.bs

```diff
@@ -284,6 +305,8 @@ An attribution trigger is a [=struct=] with the following items:
 :: An [=url/origin=].
 : <dfn>trigger data</dfn>
 :: A [=string=].
+: <dfn>event source trigger data</dfn>
```

Comment:
Per https://github.com/WICG/conversion-measurement-api/pull/202

Let's also make source_type a member on the report struct